src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Tables/Series.sql

```diff
@@ -13,25 +13,28 @@ CREATE TABLE dbo.Series (
     PartitionKey                        INT                        NOT NULL DEFAULT 1   --FK
 ) WITH (DATA_COMPRESSION = PAGE)
 
--- Ordering studies by partition, study, and series key for partition-specific retrieval
+-- Ordering studies by partition, study, and series key for partition-specific retrieval, Also used in STOW
 CREATE UNIQUE CLUSTERED INDEX IXC_Series ON dbo.Series
 (
     PartitionKey,
     StudyKey,
     SeriesKey
 )
 
-CREATE UNIQUE NONCLUSTERED INDEX IX_Series_SeriesKey ON dbo.Series
+-- used in STOW and Delete
+CREATE UNIQUE NONCLUSTERED INDEX IX_Series_PartitionKey_StudyKey_SeriesInstanceUid ON dbo.Series
 (
-    SeriesKey
+    PartitionKey,
+    StudyKey,
+    SeriesInstanceUid
 )
 WITH (DATA_COMPRESSION = PAGE)
 
--- Used in QIDO when querying at the study level; we place PartitionKey second because we assume conflicting SeriesInstanceUid will be rare
-CREATE UNIQUE NONCLUSTERED INDEX IX_Series_SeriesInstanceUid_PartitionKey ON dbo.Series
+-- Used in QIDO when querying at the study level; UIDs will not be used cross partition
+CREATE UNIQUE NONCLUSTERED INDEX IX_Series_PartitionKey_SeriesInstanceUid ON dbo.Series
```

Comment:
For QIDO at study level, the query looks like this, In case of cross partition, we will be using SeriesInstanceUID right and that needs to be on top?
```
SELECT
      x.StudyInstanceUid,
      x.SeriesInstanceUid,
      x.SopInstanceUid,
      x.Watermark
      FROM
      ( SELECT
      st.StudyKey
      ,se.SeriesKey
      FROM dbo.Study st
      INNER JOIN dbo.Series se
      ON se.StudyKey = st.StudyKey
      AND se.PartitionKey = st.PartitionKey
      WHERE 1 = 1
      AND st.PartitionKey = 1
      AND se.SeriesInstanceUid=@p0
      ORDER BY se.SeriesKey DESC
      OFFSET 0 ROWS
      FETCH NEXT 100 ROWS ONLY
      ) f
      CROSS APPLY
       (
      SELECT TOP 1
      a.StudyInstanceUid,
      a.SeriesInstanceUid,
      a.SopInstanceUid,
      a.Watermark
      FROM dbo.Instance a
      WHERE 1 = 1
      AND a.StudyKey = f.StudyKey
      AND a.SeriesKey = f.SeriesKey
      AND a.Status = 1
      ORDER BY a.Watermark DESC
      ) x
      OPTION(RECOMPILE)
```

Reply:
This looks like a series level query because of the join on the series table. It is also missing PartitionKey I added in cross apply in this PR.

I am thinking to just put PartitionKey has first column on all indexes for now, since we do not support cross partition, what do you think?

Reply:
We can support cross partition in analytics and will reduce confusion for us

Reply:
![image](https://user-images.githubusercontent.com/6980724/213618728-42b7b7cd-17fa-4c64-a4d3-c8f32b85549f.png)

Reply:
That was only old query, so missing the partitionKey in cross-apply. I am with putting partitionKey at the top now and we don't have any requirements. And we have only one customer right now using partition. But worst case, we have to rebuild the indexes later.

Reply:
if that is the case, we can update the indexes.md file too

Reply:
will do